odoo/addons/base/ir/ir_actions.py

```diff
@@ -99,9 +99,7 @@ class IrActionsReportXml(models.Model):
 
     # options
     multi = fields.Boolean(string='On Multiple Doc.', help="If set to true, the action will not be displayed on the right toolbar of a form view.")
-    attachment_use = fields.Boolean(string='Reload from Attachment', help='If you check this, then the second time the user prints with same attachment name, it returns the previous report.')
-    attachment = fields.Char(string='Save as Attachment Prefix',
-                             help='This is the filename of the attachment used to store the printing result. Keep empty to not save the printed reports. You can use a python expression with the object and time variables.')
+    post_process_report = fields.Char(string='Post Process Method')
```

Comment:
I would not use a post_process_report field. The python code calling the method can get back the hand after the report creation and it is enough I suppose